src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -4332,7 +4332,8 @@ public new View Parent
 
                 if (basehandle is Layer)
```

Comment:
Using 'Is-expressions with patterns' gives you better readability and better performance.

e.g.  if( basehandle is Layer layer )